rust/kernel/error.rs

```diff
@@ -61,6 +64,26 @@ impl Error {
     }
 }
 
+impl fmt::Debug for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // SAFETY: FFI call.
+        #[cfg(CONFIG_SYMBOLIC_ERRNAME)]
+        let name = unsafe { crate::bindings::errname(-self.0) };
+        #[cfg(not(CONFIG_SYMBOLIC_ERRNAME))]
+        let name: *const c_types::c_char = core::ptr::null();
+        if !name.is_null() {
```

Comment:
A newline to make the conditional compilation more clear.
```suggestion

        if !name.is_null() {
```